.github/workflows/bump-version-pr.yaml

```diff
@@ -0,0 +1,82 @@
+name: bump-version-pr
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_part:
+        description: "Which part of the version number to bump?"
+        required: true
+        default: "minor"
+        type: choice
+        options:
+          - "major"
+          - "minor"
+          - "patch"
+
+jobs:
+  bump-version-pr:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          fetch-depth: 0
+
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Set git config
+        uses: autowarefoundation/autoware-github-actions/set-git-config@v1
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+
+      - name: Setup Python 3.x
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: pip3 install -U catkin_tools
+        shell: bash
+
+      - name: Bump version from humble branch
+        id: bump-version-from-humble-branch
+        run: |
+          git fetch origin --tags
+          git checkout origin/humble
```

Comment:
```suggestion
```

Reply:
Thank you for suggestions!
Let me confirm. If we apply these changes, can it fetch tags simultaneously?

Reply:
I hope so but please comfirm it.

Reply:
It seems it needs commit history to merge origin/main, so I changed as like this:
https://github.com/autowarefoundation/autoware.universe/pull/9990/commits/989c0aa674c1f4bf0981fec8add01d0dbb322ef9